lib/Raven/Stacktrace.php

```diff
@@ -79,10 +79,11 @@ public static function get_stack_info($frames,
 
             // detect in_app based on app path
             if ($app_path) {
-                $in_app = (bool)(substr($abs_path, 0, strlen($app_path)) === $app_path);
+                $norm_abs_path = @realpath($abs_path) ?: $abs_path;
+                $in_app = (bool)(substr($norm_abs_path, 0, strlen($app_path)) === $app_path);
```

Comment:
@mitsuhiko we need the paths to map up to what the prefixes would look like, which happens elsewhere for $app_path, but doesnt happen for the stacktrace components. Since the $abs_path can be a non-path (or at least a path that doesnt seemingly exist as real), we handle the failure here.